src/cal/test_atmos_corr_raw_to_rad.py

```diff
@@ -278,34 +299,35 @@ def avg_intercept(total_intercept_arr, file_count):
         band7_sum += total_intercept_arr[i][6]
 
     # Computes the average intercept.
-    band1_avg = band1_sum / file_count
-    band2_avg = band2_sum / file_count
-    band3_avg = band3_sum / file_count
-    band4_avg = band4_sum / file_count
-    band5_avg = band5_sum / file_count
-    band6_avg = band6_sum / file_count
-    band7_avg = band7_sum / file_count
+    band1_avg = band1_sum / txt_count

```

Comment:
Wouldn't a numpy array make more sense? Or there is a reason these values are separated?